### PR TITLE
lynxscans

### LINKED ISSUE #15247: Lynx Scans: Updated Website
### Source information and language

Lynx Scans 1.3.6

### Steps to reproduce

Open extension
Get 404

### Expected behavior

Should load series

### Actual behavior

Gives 404

### Tachiyomi version

0.14.4

### Android version

Android 12

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #15507: Extension problem
### Source information and language

LynxScans 1.4.6

### Steps to reproduce

Opening source:
 First step: open lynxscans source
 Issue: no result found

### Expected behavior

Should display all available manga in the source

### Actual behavior

No results are returned even when there should be

### Tachiyomi version

0.14.5

### Android version

Android 12 / EMUI 12

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.5](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by henrik9999**
Closes  #15247
Closes #15507

Checklist:

- [X] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [X] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [X] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [X] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [X] Have not changed source names
- [X] Have explicitly kept the `id` if a source's name or language were changed
- [X] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/en/lynxscans/src/eu/kanade/tachiyomi/extension/en/lynxscans/LynxScans.kt [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+class LynxScans : HttpSource() {
+    override val baseUrl: String = "https://lynxscans.com"
+    private val apiUrl: String = "https://api.lynxscans.com/api"
+
+    override val lang: String = "en"
+    override val name: String = "LynxScans"
+
+    override val versionId = 2
+    override val id: Long = 2391591138348789610
```
Comment (stevenyomi) [inline src/en/lynxscans/src/eu/kanade/tachiyomi/extension/en/lynxscans/LynxScans.kt:24]:
```suggestion
```

Remove this line. The purpose of `versionId = 2` is to generate a new ID.

File: src/en/lynxscans/build.gradle [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlinx-serialization'
+
+ext {
+    extName = 'LynxScans'
+    pkgNameSuffix = 'en.lynxscans'
+    extClass = '.LynxScans'
+    extVersionCode = 1
```
Comment (stevenyomi) [inline src/en/lynxscans/build.gradle:9]:
```suggestion
    extVersionCode = 7
```

Bump this version to let users get update.

File: src/en/lynxscans/src/eu/kanade/tachiyomi/extension/en/lynxscans/LynxScans.kt [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        return MangasPage(titles, !data.chapters.next_page_url.isNullOrEmpty())
+    }
+
+    // Search
+    override fun searchMangaParse(response: Response): MangasPage {
+        throw UnsupportedOperationException("Not used")
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        throw UnsupportedOperationException("Not used")
```
Comment (stevenyomi) [inline src/en/lynxscans/src/eu/kanade/tachiyomi/extension/en/lynxscans/LynxScans.kt:61]: Make this throw a plain Exception with a proper error message. (e.g. Search is not supported)

File: src/en/lynxscans/build.gradle (hunk 2 above)
Comment (henrik9999) [reply src/en/lynxscans/build.gradle:9]: will users get an update if the extension id changes anyway?

Comment (stevenyomi) [reply src/en/lynxscans/build.gradle:9]: No. Update check is based on extension package name and version.

File: src/en/lynxscans/res/web_hi_res_512.png
Comment (mobi2002) [inline src/en/lynxscans/res/web_hi_res_512.png:1]: [you can use this for the icon](https://www.freepik.com/free-vector/terrified-girl-anime-character-poster_29108294.htm)

**Pull request merged**